src/SprykerEco/Zed/Payone/PayoneConfig.php

```diff
@@ -140,4 +140,44 @@ public function getBusinessRelation(): string
 
         return $settings[PayoneConstants::PAYONE_BUSINESS_RELATION];
     }
+
+    /**
+     * @return array
+     */
+    public function getGreenScoreAvailablePaymentMethods(): array
+    {
+        $settings = $this->get(PayoneConstants::PAYONE);
```

Comment:
Why don't you use just 
`$this->get(PayoneConstants::PAYONE_GREEN_SCORE_AVAILABLE_PAYMENT_METHODS)`

Reply:
Because Payone has strange config setup and I don't want to change this structure.